portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,132 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans:wght@300&display=swap');
+@import url('https://fonts.googleapis.com/css?family=Fira+Mono:400');
+
+/* Components */
+
+body{
+  background: white;
+  padding: 0rem 5rem;
```

Comment:
By the way, there's an HTML/CSS style guide here: https://google.github.io/styleguide/htmlcssguide.html

Main thing here is that declarations should be alphabetized. For consistency, I would also recommend adding a space between the property name and curly brace.